### ISSUE TITLE
mrcnn not valid

### ISSUE THREAD

**Issue by qayoom11** [matterport/Mask_RCNN#3023]
Hello dear @cclauss and @waleedka,

I am training my model on the mrcnn pre-trained model i.e. **mask_rcnn_coco** but unable to run successfully. After the successful model training, the following attached error is generated while testing the model. **The Tensorflow version I am using is 2.0.0**

Please help me out here.

2024-03-01 15:51:27.395494: I tensorflow/compiler/xla/service/service.cc:175] StreamExecutor device (0): Host, Default Version
Re-starting from epoch 30
1.jpg
masks (2339, 1656, 27)
2.jpg
masks (2339, 1656, 26)
3.jpg
masks (1654, 2340, 1)
4.jpg
masks (2589, 1699, 27)
5.jpg
masks (2339, 1656, 26)
6.jpg
masks (2030, 1656, 26)
7.jpg
masks (2339, 1656, 27)
8.jpg
masks (2339, 1656, 26)
mrcnn not valid
